site/content/blog/2021-10-01-whats-new-in-svelte-october-2021.md

```diff
@@ -0,0 +1,97 @@
+- [breaking] The prerender.pages config option has been renamed to prerender.entries ([#2380](https://github.com/sveltejs/kit/pull/2380))
+- A new generic argument has beend added to allow typing Body from hooks ([#2413](https://github.com/sveltejs/kit/pull/2413))
+- The "svelte" field will be added to package.json when running the package command ([#2431](https://github.com/sveltejs/kit/pull/2431))
+- [breaking] The "context" parameter of the load function was renamed to "stuff" ([#2439](https://github.com/sveltejs/kit/pull/2439))
+
+To see all updates to SvelteKit, check out the [SvelteKit changelog](https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md).
+
+
+---
+
+## Community Showcase
+
+**Apps & Sites**
+- [radiofrance](https://www.radiofrance.fr/) just migrated their website to SvelteKit
+- [FLAYKS](https://flayks.com/) is the portfolio site for Félix Péault made with SvelteKit, Sanity, and Anime.js
+- [hirehive](https://www.hirehive.com/) is an candidate and job tracking
+- [Microsocial](https://microsocial.xyz/) is an experimental Peer-to-Peer Social Platform 
+- [Dylan Ipsum](https://www.dylanlyrics.app/) is a random text generator to replace lorem ipsum with Bob Dylan lyrics
+- [Chip8 Svelte](https://github.com/mikeyhogarth/chip8-svelte) is CHIP-8 emulator frontend, built on top of CHIP8 Typescript
+
+**Looking for a Svelte project to work on? Interested in helping make Svelte's presence on the web better?** Check out [the list of open issues](https://github.com/svelte-society/sveltesociety-2021/issues) if you'd like to contribute to the Svelte Society rewrite in SvelteKit.
+
+**Podcasts Featuring Svelte**
+- [Syntax Podast: From React to SvelteKit](https://podcasts.apple.com/us/podcast/from-react-to-sveltekit/id1253186678?i=1000536276106) Scott talks with Wes about moving Level Up Tutorials from React to SvelteKit — why he did it, how, benefits, things to watch out for, and more!
+- [Web Rush Podcast: Svelte Tools and Svelte Society](https://www.webrush.io/episodes/episode-150-svelte-tools-and-svelte-society) Kevin Åberg Kultalahti talks about what Svelte Society is, what he's excited about with Svelte, how important documentation is to any product, and much _much_ more
+- [Svelte: The Compiled Future of Front End](https://www.arahansen.com/the-compiled-future-of-front-end/) details the history of component-based frontends and how a compiler changes everything
+- [Svelte Radio: Contributing to Svelte with Martin 'Grygrflzr' Krisnanto Putra](https://share.transistor.fm/s/10aa305c) Grygrflzr shares his journey to becoming a maintainer and his views on React, Vite and a host of other things
+- [Svelte Radio: Routify 3 with Jake and Willow](https://share.transistor.fm/s/10aa305c) the Svelte Radio crew sits down the maintainers of Routify and talk about the just-released Routify 3
```

Comment:
```suggestion
- [Svelte Radio: Routify 3 with Jake and Willow](https://share.transistor.fm/s/10aa305c) the Svelte Radio crew sits down with the maintainers of Routify and talks about the just-released Routify 3
```

Reply:
Should definitely add the "with"

I'm less certain about changing "talk" to "talks". The other option there would be to change "sits" to "sit". I actually had to Google this as a native speaker because I couldn't figure out which was right. According to [the post I found](https://stancarey.wordpress.com/2017/03/16/is-the-crew-plural-collective-complications/) it would be correct to say either "sit"/"talk" or "sits"/"talks", but they must be consistent and the current wording is wrong because "sits"/"talk" does not consistently treat crew as singular or plural. Personally, "sit"/"talk" sounds most natural to me, but either way would be fine